ci-operator/step-registry/ipi/conf/gcp/osdisk/disksize/ipi-conf-gcp-osdisk-disksize-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-patch.yaml"
+
+BASE_SIZEGB=256
+if [ -z "${COMPUTE_DISK_SIZEGB}" ]; then
+  COMPUTE_DISK_SIZEGB=$(( $BASE_SIZEGB * (RANDOM % 3 + 1) ))
```

Comment:
Why need a RANDOM value here?

Reply:
To avoid using a fixed disk size for compute and control-plane nodes, if the env vars not specified. Your suggestion?

Reply:
Actually I do not suggest to do many silent customization in the common gcp-ipi install workflow, once something is breaking the install, it will affect other workflows too which reference this provision chain. 

Another point is we are trying our best to follow a rule, reflecting the setting in the install workflow, so if we think inserting this step in this chain is a good choice, then maybe we should exit it directly when `COMPUTE_DISK_SIZEGB` and `CONTROL_PLANE_DISK_SIZEGB` are empty, in order to avoid introducing unexpected change on this base install workflow.

Reply:
> To avoid using a fixed disk size for compute and control-plane nodes

I think we should leave that to users to decide to set them via `COMPUTE_DISK_SIZEGB` and `CONTROL_PLANE_DISK_SIZEGB`